methods/build.js

```diff
@@ -58,23 +61,11 @@ module.exports = function (config, settings, data) {
         before
       });
     }
-    if (item.hooks && !noDiff) {
-      try {
-        await server.methods.processHooks(item);
-      } catch (e) {
-        const err = (e.output) ? { message: `Error: ${item.image} hook error`, output: e.output } : { message: `Error: ${item.image} hook error`, err: e };
-        server.log(['docker-autobuild', 'hook', 'error'], err);
-      }
-    }
-  };
-
-  const buildServices = async function(list) {
-    for (const d of list) {
-      await buildService(d); // eslint-disable-line no-await-in-loop
-    }
   };
 
-  buildServices(config);
+  await pMap(config, (d) => {
```

Comment:
actually, this changed the functionality of this - before it would do it serially and now it does it in parallel.  Also, we didn't wait for it to finish to return a response.  Builds can take a long time, so I wouldn't want to wait for it to finish to return a response